k8s/jenkins/chart/jenkins/templates/application.yaml

```diff
@@ -30,43 +30,38 @@ spec:
       url: https://jenkins.io/doc/
     notes: |-
       # Prerequisites
-
       You'll need the following tools in your environment:
 
       *  [gcloud](https://cloud.google.com/sdk/gcloud/)
+        *  If gcloud has not been configured yet, then configure gcloud by following the [gcloud Quickstart](https://cloud.google.com/sdk/gcloud/).
       *  [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+        *  Set kubectl to a specific cluster by following the steps at [container get-credentials](https://cloud.google.com/sdk/gcloud/).
       *  [sed](https://www.gnu.org/software/sed/)
 
-      # Optional step, only if gcloud and kubectl are not configured yet.
-
-      First you need to configure gcloud, refer to [gcloud Quickstart](https://cloud.google.com/sdk/gcloud/) and
-      set kubectl to use specific cluster, refer to [container get-credentials](https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials)
-
-      # Access Jenkins instance.
-
-      Identify HTTPS endpoint.
+      # Accessing your Jenkins instance
+      Identify the HTTPS endpoint:
 
       ```shell
       echo https://$(kubectl -n{{ .Release.Namespace }} get ingress -l "app.kubernetes.io/name={{ .Release.Name }}" -ojsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")/
       ```
 
-      For HTTPS you have to accept a certificate (we created a temporary one for you). Now you need a password.
+      For HTTPS, you must accept a certificate (we have created a temporary one for you). You will also need a password:
```

Comment:
What do you think about moving "For HTTPS, you must accept a certificate (we have created a temporary one for you)." just after "Accessing your Jenkins instance" header?

For example:

```
# Accessing your Jenkins instance

> NOTE: For HTTPS, you must accept a temporary TLS certificate.

1. Read a temporary password:

    ```shell
    kubectl -n{{ .Release.Namespace }} exec \
    $(kubectl -n{{ .Release.Namespace }} get pod -oname | sed -n /\\/{{ .Release.Name }}-jenkins/s.pods\\?/..p) \
    cat /var/jenkins_home/secrets/initialAdminPassword
    ```

1. Identify the HTTPS endpoint:

    ```shell
    echo https://$(kubectl -n{{ .Release.Namespace }} get ingress -l "app.kubernetes.io/name={{ .Release.Name }}" -ojsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")/
    ```

1.  Navigate to the endpoint.
```